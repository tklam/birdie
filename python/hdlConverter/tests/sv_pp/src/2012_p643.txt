`define max(a,b)((a) > (b)) ? (a) : (b)
n = `max(p+q, r+s) ;
