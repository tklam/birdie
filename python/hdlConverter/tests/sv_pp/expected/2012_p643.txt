
n = ((p+q) > (r+s)) ? (p+q) : (r+s) ;
